api/src/modules/users/users.controller.ts

```diff
@@ -1,29 +1,42 @@
-import { Controller, Delete, Get, Param, Post, Put } from '@nestjs/common';
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  Param,
+  Post,
+  Put,
+} from '@nestjs/common';
+import { UsersService } from './users.service';
+import { User } from '@shared/dto/users/user.entity';
+import { CreateUserDto } from '@shared/dto/users/create-user.dto';
 
-@Controller()
+@Controller('users')
 export class UsersController {
-  @Post('users')
-  create(): string {
-    return 'This action adds a new user';
+  constructor(private usersService: UsersService) {}
+
+  @Post()
+  async save(@Body() createUserDto: CreateUserDto) {
+    this.usersService.save(createUserDto);
```

Comment:
It's not wrong, but let's be consistent with the rest of operations and `return` the outcome of the service.

Reply:
It is actually wrong, as you are neither awaiting for the process to kick back from the even loop, neither returning any value for the same purpose. 

You could potentially run into race conditions like this